lib/core/base/context/parse-selector-array.js

```diff
@@ -12,29 +12,20 @@ export function parseSelectorArray(context, type) {
   const result = [];
   for (let i = 0, l = context[type].length; i < l; i++) {
     const item = context[type][i];
-    // selector
-    if (typeof item === 'string') {
```

Comment:
This cannot happen. After normalising, this is always either a node, or an array..